### ISSUE TITLE
Followed "Milestone 2" release tutorial. End up with `Window` with red underline

### ISSUE THREAD

**Issue by ColtonIdle** [JetBrains/compose-jb#173]
I followed the tutorial here:
https://blog.jetbrains.com/cross-post/jetpack-compose-for-desktop-milestone-2-released/

but I ended up with a red underline under `Window`
<img width="930" alt="Screen Shot 2020-12-04 at 7 22 12 PM" src="https://user-images.githubusercontent.com/31751141/101331650-4a458880-3842-11eb-99f4-1ca8cdc1667f.png">

Using Intellij 2020.3 community edition. Can repro every time I create a new project. 
<img width="583" alt="Screen Shot 2020-12-07 at 4 13 15 AM" src="https://user-images.githubusercontent.com/31751141/101332019-c049ef80-3842-11eb-8bc2-5381faf2b07b.png">
<img width="566" alt="Screen Shot 2020-12-07 at 4 13 33 AM" src="https://user-images.githubusercontent.com/31751141/101332021-c049ef80-3842-11eb-9c83-0eecd610144d.png">

**Comment by GGorAA** [comment]
Try JDK 15
You can download it using IntelliJ

**Comment by ColtonIdle** [comment]
According to the tutorial java 11 is fine?

>Note that JDK must be at least JDK 11, and to use the native distribution packaging JDK 14 or later must be used.
https://github.com/JetBrains/compose-jb/tree/master/tutorials/Getting_Started

**Comment by GGorAA** [comment]
idk
I use jdk 15, no problems for now
you can just try

**Comment by igordmn** [comment]
If we hover over Window, what error will IDEA show?

Also can you please try on clean installation of IDEA, without importing old settings?

Or make sure that these settings are default:
![image](https://user-images.githubusercontent.com/5963351/101353050-0be0b580-38a4-11eb-9033-0bad633ab66a.png)
![image](https://user-images.githubusercontent.com/5963351/101353077-14d18700-38a4-11eb-9559-fbdc027b2b61.png)

Also version of Kotlin plugin in IDEA may be not 1.4.20. Check here:
![image](https://user-images.githubusercontent.com/5963351/101353224-4a767000-38a4-11eb-9bc2-1c068de057f0.png)

**Comment by ColtonIdle** [comment]
> If we hover over Window, what error will IDEA show?

<img width="836" alt="Screen Shot 2020-12-07 at 12 19 26 PM" src="https://user-images.githubusercontent.com/31751141/101382973-90bcd680-3886-11eb-8817-884cbbac1c14.png">


> Also can you please try on clean installation of IDEA, without importing old settings?

This was a clean install! =) I installed IDEA just to try compose desktop!

> Or make sure that these settings are default:

Just double checked everything and everything looks good EXCEPT the kotlin plugin is set to 1.4.10 apparently? But that's the latest on stable available to me???

<img width="495" alt="Screen Shot 2020-12-07 at 12 22 28 PM" src="https://user-images.githubusercontent.com/31751141/101383322-f9a44e80-3886-11eb-86be-dd38733527e6.png">

**Comment by GGorAA** [comment]
Update to the latest IDEA(if you are on an outdated one), and then check for updates

**Comment by GGorAA** [comment]
Latest Kotlin Plugin 1.4.20 is only compatible with IntelliJ IDEA 2020.3

**Comment by ColtonIdle** [comment]
I'm on latest stable IDEA

<img width="686" alt="Screen Shot 2020-12-07 at 12 25 33 PM" src="https://user-images.githubusercontent.com/31751141/101383617-5a338b80-3887-11eb-87c0-b0d0977d0edf.png">

**Comment by igordmn** [comment]
>If we hover over Window, what error will IDEA show?

BufferedImage was in the very old version of Java. Defenetely before Java 1.6.
If we just import java.awt.image.BufferedImage, will IDEA highlight it?

>is set to 1.4.10 apparently

It seems for 2020.3 the plugin is only in Early Access:
![image](https://user-images.githubusercontent.com/5963351/101384670-c637e300-38cb-11eb-9f16-74c912992cd9.png)

I also checked on 1.4.10 and everything was fine too (and on Java 11.0.9).

**Comment by ColtonIdle** [comment]
@igordmn 

> BufferedImage was in the very old version of Java. Defenetely before Java 1.6.
> If we just import java.awt.image.BufferedImage, will IDEA highlight it?

Nope. It doesn't like the import at all. "Unresolved reference java"

> It seems for 2020.3 the plugin is only in Early Access:

So do you want me to try the early access 1.4.20 plugin? I'm able to repro this issue every time I do file > new project so I figured my bug report could help those that are new to desktop compose because it seems like a real error. @igordmn I'm guessing you can't repro?

**Comment by ColtonIdle** [comment]
I opened up the build.gradle file for the first time... red everywhere.

Something definitely
<img width="826" alt="Screen Shot 2020-12-07 at 7 47 56 PM" src="https://user-images.githubusercontent.com/31751141/101422667-3858f980-38c5-11eb-8350-e1ba4a9ebaa7.png">
 is not right.

**Comment by olonho** [comment]
Generally looks like import problem. Could you please recreate fresh project from scratch and see if the problem persists?

**Comment by igordmn** [comment]
>  @igordmn I'm guessing you can't repro?

Yes, everything is fine on my configuration.

Can you try to sync the project?
![image](https://user-images.githubusercontent.com/5963351/101455724-13579b80-3944-11eb-833f-8dcb36ce511d.png)

After the sync will IDEA show any errors on "Problems" tab and "Build" tab?
![image](https://user-images.githubusercontent.com/5963351/101455802-29fdf280-3944-11eb-8cd2-e13fe2019980.png)

**Comment by ColtonIdle** [comment]
@igordmn something is definitely wrong on my end then.

<img width="826" alt="Screen Shot 2020-12-08 at 10 22 07 AM" src="https://user-images.githubusercontent.com/31751141/101502729-4cd9d800-393f-11eb-8b04-ae164f54fe59.png">

I am able to recreate the issue every time.

Question: How do you start the project? I just hit the green arrow next to `fun main()`

**Comment by igordmn** [comment]
Oh, sorry, I forgot that this is my custom button :D.

There is a default button:
![image](https://user-images.githubusercontent.com/5963351/101503608-52f1a800-3983-11eb-8a21-0dcf97580329.png)

> How do you start the project? I just hit the green arrow next to fun main()

I run as a gradle task:
![image](https://user-images.githubusercontent.com/5963351/101503890-a82db980-3983-11eb-87c7-704662edc3d5.png)

But if I hit the green arrow next to `main` everything works too.

**Comment by ColtonIdle** [comment]
Same issue

Here is a gif of a new project
![Kapture 2020-12-08 at 10 37 55](https://user-images.githubusercontent.com/31751141/101505332-3d0fc300-3942-11eb-89ac-a05b241567a9.gif)

**Comment by igordmn** [comment]
>Same issue
After the sync is IDEA showing any errors on tabs "Problems" and "Build"?

>I opened up the build.gradle file for the first time... red everywhere.
If everything is red in gradle file then IDEA doesn't properly sync with Gradle and should report about that (in theory)

**Comment by igordmn** [comment]
Also, will project run without IDEA?

We need to define JAVA_HOME variable and run from terminal:
```
./gradlew run
```

**Comment by ColtonIdle** [comment]
After hitting the gradle sync button, this is my problem pane output

<img width="1360" alt="Screen Shot 2020-12-08 at 11 00 04 AM" src="https://user-images.githubusercontent.com/31751141/101507509-a0025980-3944-11eb-97a1-2765d41b5cc9.png">

I just want to make it clear. **The project runs just fine, its just the red lines in the IDE that are worrying me.** 

So yes, hitting the run button by main starts the desktop app, so does hitting run from the gradle tasks pane, so does ./gradlew run.

**Comment by igordmn** [comment]
Probably something wrong with JDK or how JDK is registered in IDEA.

Try to download JDK from here:
https://jdk.java.net/archive

And try to clear all JDK's from here and add JDK 11 again:
![image](https://user-images.githubusercontent.com/5963351/101512677-53db0780-398c-11eb-895e-7b9a478870f5.png)

I assume non-compose projects have the same issue?

**Comment by ColtonIdle** [comment]
Non compose projects don't have the same issue.

But you are right! The JDK is the issue.

I have two JDK 11's.

**Java 11:**
/Users/coltonidle/Library/Application Support/JetBrains/Toolbox/apps/IDEA-C/ch-0/203.5981.155/IntelliJ IDEA CE.app/Contents/jbr/Contents/Home

<img width="1283" alt="Screen Shot 2020-12-08 at 2 29 39 PM" src="https://user-images.githubusercontent.com/31751141/101531948-dc907e00-3961-11eb-9e8f-9453c1fd764f.png">

**Java 11 (2):**
/Library/Java/JavaVirtualMachines/adoptopenjdk-11.jdk/Contents/Home


**If I set it to my adoptopenjdk then everything works fine.**

<img width="1276" alt="Screen Shot 2020-12-08 at 2 29 00 PM" src="https://user-images.githubusercontent.com/31751141/101531980-e74b1300-3961-11eb-9d5f-4955f4519fea.png">

**Do you think this is a bug I should file against the jdk packaged into IntelliJ?**

**Comment by igordmn** [comment]
On macOs and this JDK everything is fine too on my configuration 🤔.

But it is internal custom JDK and shouldn't be used for applications (only for running IDEA itself):
https://confluence.jetbrains.com/display/JBR/JetBrains+Runtime
"JetBrains Runtime is not a certified build of OpenJDK. Please, use at your own risk."

> Do you think this is a bug I should file against the jdk packaged into IntelliJ?
Thanks! But I think if everything is fine with adoptopenjdk, I don't think it is a bug.

Did you add this JDK manually or was it automatically added into SDK's?

**Comment by ColtonIdle** [comment]
I don't think I did anything special to add it. It's set as my java_home and so I think it was added by itself.

I did not realize that you should not use the packaged in IJ jdk for a project. A warning would be nice, but I agree that it doesn't answer the question of why it's working for you. I might try on my other machine and see what happens later today and I'll let you know. If it happens on my other machine then maybe it's worth digging into a little deeper.

**Comment by ColtonIdle** [comment]
Alright. I tried on another old machine I had. Installed latest intellij and basically what I saw was that the JDK that ships with intellij was not part of the list and so there was no opportunity for me to accidently select it over my java_home jdk.

So I think we're in the clear. Hopefully this helps someone else in the future that might have (accidently) had the integrated jdk set as a valid jdk in the ide.

Thanks @igordmn !!!
